src/index.jsx

```diff
@@ -7,6 +7,7 @@ import HallOfFamePage from "./Pages/HallOfFamePage/index";
 import GlobalStyle from "./Styles/GlobalStyle";
 import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
 import PaintBoard from "./Pages/PaintBoard";
+import ThreadPictureNullPage from "./Pages/ThreadPictureNullPage";
```

Comment:
page 폴더에 Barrel 파일을 만들어 모든 페이지를 한 번에 Import 해오는 건 어떨까요?
```suggestion
import * as P from "./Pages";
```

Reply:
a1d44b336d1bf50b324b34479fa35432535ec62f 수정 하였습니다!